### ISSUE TITLE
A way to see long filenames

### ISSUE THREAD

**Issue by embryo10** [koreader/koreader#2260]
* KOReader version: 2015.11-587
* Device: Kobo Glo

#### Issue
While browsing my books with the file browser, it would be nice to have a way to see long filenames.
The files of my books are formatted as 'Author' - 'series name' 'series nr' 'title'.
So a name like "Larry Niven & Edward M. Lerner - Fleet of Worlds 02 Juggler of Worlds.epub" is impossible to view and impossible to distinguish from the other books in the same series. 
The same is true for the History dialog.
There, it gets more complicated since the order is always changing depending on the last opened time.
A workaround (not a full solution) would be to have a way to change to Landscape view, or a way to decrease/increase the font size.
A better solution would be to have a way of horizontal scrolling (with steps of half a page to the right/left)

edit: re-checked with 2015.11-587, no change.

**Comment by AlanSP1** [comment]
This would be useful. I have some very long file names and being able to see them completely would be nice.

**Comment by KenMaltby** [comment]
For me, with my Kobo AuraHD, it is in .adds\koreader\frontend\ui\widget\menu.lua

 The "cfont", at line 122 and 325, using Notepad++

 I changed mine to 18. It looks fine but The Aura HD, has a higher res than some, which means that the fine print is clear and readable. The text is smaller, so more of the titles are visible.  16 works,as well.

To adjust number of lines per page, change:
 Around Line 374: Screen:scaleBySize(46), -- hardcoded for now. I changed to (28), for my AuraHD)
---------------------------------------
\koreader\frontend\apps\reader\modules\readerfooter.lua Line 251 return os.date ("%I:%M") for 12 hour format on the mini-bar.
